packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -76,9 +108,64 @@ export class ConfigUtil {
     return undefined;
   }
 
-  public static async getUserConfiguredApiVersion() {
-    const apiVersion = await ConfigUtil.getConfigValue('apiVersion');
-    return apiVersion ? String(apiVersion) : undefined;
+  public static async getUserConfiguredApiVersion(): Promise<
```

Comment:
nit: same as https://github.com/forcedotcom/salesforcedx-vscode/pull/4370/files#r949573633
Why do we have the user api version method here and the org api method over in authInfo. 

Should we maybe have an API specific service or is that too granular?

Reply:
This one is a configuration value that is set by the user. It is intended to be used to override the other one in some cases, which is the API version of the currently authenticated Org. So I think they are in the correct places.

Reply:
would this info be worth a method comment?